src/app/api/tools/util.ts

```diff
@@ -79,7 +79,12 @@ export const getAlchemyRpcUrl = (chainId: number): string => {
 };
 
 export const getClientWithAlchemy = (chainId: number) => {
+  const chain = ALCHEMY_CHAINS.find((c) => c.id === chainId);
   return createPublicClient({
+    batch: {
+      multicall: true,
```

Comment:
Are we sure this is necessary? I have never needed to specify this.

Reply:
Right... but we can use multicall even without specifying it.

I did it here and never needed to specify this:

https://github.com/BitteProtocol/core/pull/52/files#r2205932655